src/frontend/screens/WebView/index.tsx

```diff
@@ -225,7 +225,10 @@ export default function WebView() {
         if (validatedURL && validatedURL.match(/track\.adtraction\.com/)) {
           const parsedUrl = new URL(validatedURL)
           const redirectUrl = parsedUrl.searchParams.get('url')
-          webview.loadURL(redirectUrl || 'https://gog.com')
+          const url = new URL(redirectUrl || 'https://gog.com')
+          // Remove any port definitions, they are dumb
```

Comment:
Maybe add the explanation you provided in the issue here? Just "they are dumb" might be humorous, but I imagine a confused me looking at this in about a year going "why is this there again?"